modules/steam/hm.nix

```
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.stylix.targets.steam;
+in {
+  options.stylix.targets.steam = {
+    enable = config.lib.stylix.mkEnableTarget "Steam" true;
+    adwaitaForSteam.enable = config.lib.stylix.mkEnableTarget "Adwaita for Steam" false;
+  };
+
+  config = lib.mkIf (config.stylix.enable && cfg.enable) (lib.mkMerge [
+    {
+      # Generic configuration
+    }
+
+    (lib.mkIf cfg.adwaitaForSteam.enable {
+      home.packages = with pkgs; [adwsteamgtk];
+
+      home.activation.adwaitaForSteam = let
+        shellScript = pkgs.writeShellScript "adwaitaForSteam" ''
+          rm -f "$HOME/.cache/AdwSteamInstaller/extracted/custom/custom.css"
+          ${lib.getExe pkgs.adwsteamgtk}  -i
```

Consider removing the duplicate space and using the long flag name:

```suggestion
          ${lib.getExe pkgs.adwsteamgtk} --install
```